proto/proto/strapper.proto

```diff
@@ -10,9 +10,17 @@ message Interface {
 	repeated string ipaddr = 3;
 }
 
+message PublicKeys {
+	string rsa = 1;
+	string dsa = 2;
+	string ecdsa = 3;
+	string ed25519 = 4;
+}
+
 message NodeAdvertisement {
 	string hostname = 1;
 	repeated Interface interfaces = 2;
+	PublicKeys pubkeys = 3;
```

Comment:
As discussed, this should likely just be a map<string, string> to allow for flexible algorithms

Reply:
Also these are fingerprints, not the pubkeys themselves. ssh_fingerprints?